purchase_request/models/purchase_request.py

```diff
@@ -86,6 +86,9 @@ def _compute_is_editable(self):
                                readonly=False,
                                copy=True,
                                track_visibility='onchange')
+    product_id = fields.Many2one('product.product',
+                                 related='line_ids.product_id',
+                                 string='Product')
```

Comment:
@aheficent Careful here. When you will write on purchase request, you will generate a write also on product.
`readonly=True`

Reply:
Is it not readonly by deafult? Odoo does the same in purchases: https://github.com/odoo/odoo/blob/11.0/addons/purchase/models/purchase.py#L155

Reply:
Introduced in v12. Odoo has also wrong code :smile:

Reply:
v11 : https://github.com/odoo/odoo/blob/11.0/odoo/fields.py#L413

v12 : https://github.com/odoo/odoo/blob/12.0/odoo/fields.py#L422

Reply:
Fix proposed: https://github.com/OCA/purchase-workflow/pull/648